src/main/resources/ToolBar.fxml

```diff
@@ -48,7 +48,7 @@
                                         <ImageView image="$gridBtn"/>
                                     </graphic>
                                 </ToggleButton>
-                                <Spinner fx:id="spacingSize" amountToStepBy="1.0" editable="true" initialValue="5.0"
+                                <Spinner fx:id="snapGridSize" amountToStepBy="1.0" editable="true" initialValue="5.0"
```

Comment:
Same as comment in this context, it will be changed with the rulers feature. pls do not change